lib/openapi/mongoid/spec_builder.rb

```diff
@@ -409,6 +417,14 @@ def build_crud_specification(routes)
                       key :'$ref', name
                     end
                   end
+
+                  response 422 do
+                    key :description, 'UnprocessableEntityError'
```

Comment:
Maybe we could use spaces between words here? as in https://github.com/slate-studio/openapi-rails/pull/9/files#diff-df6ce6fd1e5038e765331f693008d7e0L388